modules/data-table/main/index.coffee

```diff
@@ -677,6 +675,8 @@ class DataTable extends hx.EventEmitter
         feed.rows range, ({rows, filteredCount}) =>
           if options.displayMode is 'paginate'
             multiPage = false
+            selection.classed('hx-data-table-infinite', filteredCount is undefined)
```

Comment:
This is to prevent a seemingly orphaned "of <n>" shown without the pageSelector next to it.